features/buyer/direct-award.feature

```diff
@@ -1,7 +1,103 @@
 @direct-award
 Feature: Direct Award flows
+  
+Scenario: Unauthenticated user can save a search after logging in
+  Given I have a production buyer user
+  And I am on the /g-cloud/search page
+  And I click 'Save search'
+  Then I am on the 'Log in to the Digital Marketplace' page
+  When I enter that user.emailAddress in the 'Email address' field
+  And I enter that user.password in the 'Password' field
+  And I click the 'Log in' button
+  Then I see the 'Log out' link
+  Then I am on the 'Choose where to save your search' page
+  And I choose the 'Create a new saved search' radio button
+  And I enter 'my cloud project' in the 'Name your search' field
+  And I click 'Save and continue'
+  Then I am on the 'my cloud project' page
+
+Scenario: User can save a search
+  Given I am logged in as a buyer user
+  And I am on the /g-cloud/search page
+  And I click 'Save search'
+  Then I am on the 'Choose where to save your search' page
+  And I enter 'my cloud project' in the 'Name your search' field
+  And I click 'Save and continue'
+  Then I am on the 'my cloud project' page
+
+Scenario: User with saves searches completes new saved search
+  Given I am logged in as a buyer user
+  And I have created and saved a search
+  And I am on the /g-cloud/search page
+  And I click 'Save search'
+  Then I am on the 'Choose where to save your search' page
+  And I choose the 'Create a new saved search' radio button
+  And I enter 'my cloud project' in the 'Name your search' field
+  And I click 'Save and continue'
+  Then I am on the 'my cloud project' page
+
+Scenario: User updates existing saved search
+  Given I am logged in as a buyer user
+  And I have created and saved a search
+  And I am on the /g-cloud/search page
+  And I click 'Save search'
+  Then I am on the 'Choose where to save your search' page
+  And I choose the 'my cloud project' radio button
+  And I click 'Save and continue'
+  Then I am on the 'my cloud project' page
 
-Scenario: User can save a search into a new Direct Award Project
+Scenario: User edits existing search
   Given I am logged in as a buyer user
   And I have created and saved a search
+  And I am on the /buyers page
+  Then I click the 'View your saved searches' link
+  Then I click the 'my cloud project' link
+  Then I am on the 'my cloud project' page
+  And I click the 'Edit search' link
+  Then I am on the 'Search results' page
+  And I click 'Save search'
+  Then I am on the 'Choose where to save your search' page
+  And I choose the 'my cloud project' radio button
+  And I click 'Save and continue'
+  Then I am on the 'my cloud project' page
+
+Scenario: User ends search and downloads results
```

Comment:
As all the steps here are duplicated in the following Scenario I think we can drop this first one altogether, and rename the following one as something like "User can end their search, download results and return to re-download them later."

Reply:
the first one the user doesn't have any saved searches, the second one the user has existing searches

Reply:
I can't see a difference - all the scenario steps are exactly the same, so all the data setup must be the same? i.e. the 14 steps of `Scenario: User ends search and downloads results` are the **exact same 14 steps** at the start of `Scenario: User redownloads results` - so these 14 steps are performed twice exactly the same.

Reply:
Hi @TheDoubleK - I have merged the two together.